tests/baselines/reference/systemModule11.js

```diff
@@ -1,21 +1,113 @@
-//// [systemModule11.ts]
+//// [tests/cases/compiler/systemModule11.ts] ////
+
+//// [file1.ts]
+
+// set of tests cases that checks generation of local storage for exported names
+
+
+export var x;
+export function foo() {}
+export * from 'bar';
+
+//// [file2.ts]
 
-import 'foo'
-import {f} from 'bar';
+var x;
+var y;
+export {x};
+export {y as y1}
 
-f();
+export * from 'bar';
 
-//// [systemModule11.js]
-System.register(['foo', 'bar'], function(exports_1) {
-    var bar_1;
+//// [file3.ts]
+
+export {x, y as z} from 'a';
+export * from 'bar';
+
+//// [file4.ts]
+
+export var x;
+export function foo() {}
+
+var z, z1;
+export {z, z1 as z2};
+
+export {s, s1 as s2} from 'a'
+
+//// [file1.js]
+// set of tests cases that checks generation of local storage for exported names
+System.register(['bar'], function(exports_1) {
+    var x;
+    function foo() { }
+    exports_1("foo", foo);
+    var exportedNames_1 = { 
+        'x': true,
+        'foo': true
+    };
+    return {
+        setters:[
+            function (_bar_1) {
+                for (var n in _bar_1)
+                    if (!exportedNames_1.hasOwnProperty(n)) exports_1(n, _bar_1[n]);

```

Comment:
per spec (15.2.1.16.3 ResolveExport( exportName, resolveSet, exportStarSet ) Concrete Method) if some name can be resolved from more than one star export the result is ambiguous, When re-exports and imports are resolved during module instantiation it is a syntax error to have ambiguous resolution (per 15.2.1.16.4 ModuleDeclarationInstantiation( ) Concrete Method).Since this situation is an error (not sure if ES6 module loader treats it as such) I think it is ok to keep this aspect implementation specific.